examples/solids/qfunctions/finite-strain-neo-hookean-initial-1-ad.h

```diff
@@ -0,0 +1,710 @@
+// Copyright (c) 2017, Lawrence Livermore National Security, LLC. Produced at
```

Comment:
We can cut out the completely repeated functions (energy, diagnostic) and use both this header and `finite-strain-neo-hookean-initial-1.h` in `finite-strain-neo-hookean-initial-1-ad.c`

Reply:
How about [this approach](https://github.com/CEED/libCEED/pull/834/commits/764d88583e092121b5e5bf778d417cebf47926ec)?